src/Components/GraduatesList/GraduatesList.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Checkbox, Input, Table } from "semantic-ui-react";
+import FilterButton from "./FilterButton";
+import Graduated from "./Graduated";
+import axios from "axios";
+
+class GraduatesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      graduates: []
+    };
+  }
+
+  componentDidMount() {
+    const API_URL = process.env.REACT_APP_API_URL;
+    axios
+      .get(`${API_URL}/empresas-egresades`)
+      .then(response => {
+        this.setState({
+          graduates: response.data
+        });
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  }
+
+  listGraduates = () => {
+    return this.state.graduates.map((currentList, index) => {
+      return <Graduated item={currentList} key={index} />;
+    });
+  };
+
+  render() {
+    return (
+      <div>
```

Comment:
Using **div** could cause responsive problems

Reply:
For this sprint, responsive is not part of our scope